src/testimonials/testimonials.controller.ts

```diff
@@ -33,7 +33,7 @@ import { TestimonialsService } from './testimonials.service';
 export class TestimonialsController {
   private readonly logger = new Logger(TestimonialsController.name);
 
-  constructor(private readonly testimonialsService: TestimonialsService) {}
```

Comment:
Please remove irrelevant change

Reply:
It was an automatic linter fix. Removed.